R/calcIndustry_EEK.R

```diff
@@ -44,7 +49,7 @@ calcIndustry_EEK <- function(kap) {
     madrat_mule()
 
   ## industry subsector activity and FE projections ----
-  FEdemand <- calcOutput(type = 'FEdemand', aggregate = FALSE, supplementary = FALSE)
+  FEdemand <- calcOutput("FEdemand", scenario = unique(c("SSP2", scenario)), aggregate = FALSE)
```

Comment:
Switching this to `unique(c(scenario, "SSP2"))` will present `madrat` with identical arguemnts, leveraging the existing cache.